wasm2c/README.md

```diff
@@ -327,6 +349,23 @@ arguments and returning `void` . e.g.
 
 `wasm_rt_free_memory` frees the memory instance.
 
+`wasm_rt_allocate_memory_shared` initializes a memory instance that can be
+shared by different Wasm instances, and allocates at least enough space for the
```

Comment:
Should this be "shared by different Wasm threads" (or "agents") rather than "instances"? Even a normal memory can be shared by different module instances.